package/resources/data/order_type.xml

```diff
@@ -12,4 +12,22 @@
         <field name="name">Radiology Order</field>
         <field name="description">Radiology orders are used to request imaging studies such as X-rays, CT scans, MRI scans, and ultrasound examinations for patients. These orders provide details about the type of imaging study needed, the body part to be examined, and any relevant clinical information.</field>
     </record>
+    <record id="order_type_shop_map_drug" model="order.type.shop.map">              
+        <field name="order_type" ref="order_type_drug"/>
+        <field name="shop_id" ref="sale_shop_pharmacy"/>
+        <field name="location_name">General Ward</field>
```

Comment:
I feel that we should not hardcode location names for the default mapping. Lets go with shop mapping without location, if a specific mapping is needed let the implementer make the manual mapping. This will make this initizlizer very specific to the current OpenMRS db backup

Reply:
@mohan-13 As discussed, the data file here is for testing how the basic metadata is initialized. This set is based on our current Bahmni setup. Once the epic is fully tested, we can remove these files from the image and mount them back in. Since these configurations can be adjusted by the implementer, we can leave them as is for now, allowing the implementer to customize the location_name according to their needs.